templates/project-harbor/2/rancher-compose.yml

```diff
@@ -0,0 +1,51 @@
+version: '2'
+.catalog:
+  name: Harbor
+  version: 1.1.1-singlehost-rev0
```

Comment:
In order to 1.1.1-1-rev1 are the last version and to avoid "update available" message...Could you please change version name??
```
--  version: 1.1.1-singlehost-rev0
++  version: 1.1.1-1-rev0-singlehost
```